modules/pulsar/src/main/java/org/testcontainers/containers/PulsarContainer.java

```diff
@@ -25,6 +25,11 @@ public PulsarContainer(String pulsarVersion) {
         waitingFor(Wait.forHttp(METRICS_ENDPOINT).forStatusCode(200).forPort(BROKER_HTTP_PORT));
     }
 
+    public PulsarContainer withFunctionsWorker() {
```

Comment:
I'd be better to store it as a `boolean` flag and decide which command to use in `configure`. Also, I'd use a composite wait strategy that will wait for both Pulsar's readiness and that the Function worker has started